core/src/main/java/org/commonjava/maven/ext/core/impl/RelocationManipulator.java

```diff
@@ -209,7 +209,7 @@ private boolean updateDependencies( WildcardMap<ProjectVersionRef> relocations,
                     // Unfortunately because we iterate using the resolved project keys if the relocation updates those
                     // keys multiple iterations will not work. Therefore we need to remove the original key:dependency
                     // to map to the relocated form.
-                    postFixUp.put( SimpleArtifactRef.parse( dependency.getManagementKey() ), dependency );
+                    postFixUp.put( new SimpleScopedArtifactRef( dependency ), dependency );
```

Comment:
Is there some reason for changing this or are the two statements equivalent? I see the parse version used later in another file.

Reply:
`getManagementKey` returns something that would not be parsable - its missing a version. By using the original object that avoids potential parse issues.